docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -57,3 +57,72 @@ ServerConfiguration.builder()
 ----
 <1> The name of the application (service) to associate with the tracing events
 <2> The endpoint for tracing events, specific to the tracer used, usually loaded from Config
+
+=== Configuring Tracing with Helidon MP
+There is an additional dependency that adds Tracing support to MP server and Jersey client:
```

Comment:
The first sentence should describes the "tracing support to MP server and Jersey client".
Then you can just add a simple sentence in a separate paragraph like "Declare the following dependency in your project".

Reply:
Fixed

Reply:
Reworded